tree/ntuple/v7/src/RColumnElement.cxx

```diff
@@ -36,6 +37,7 @@ ROOT::Experimental::Detail::RColumnElementBase::Generate<void>(EColumnType type)
    case EColumnType::kBit: return std::make_unique<RColumnElement<bool, EColumnType::kBit>>();
    case EColumnType::kReal64: return std::make_unique<RColumnElement<double, EColumnType::kReal64>>();
    case EColumnType::kReal32: return std::make_unique<RColumnElement<float, EColumnType::kReal32>>();
+   case EColumnType::kReal16: return std::make_unique<RColumnElement<float, EColumnType::kReal16>>();
```

Comment:
Let's perhaps add a TODO to change this to `f16` in-memory type once it is available.